src/main/kotlin/lotto/view/LottoResultMessage.kt

```diff
@@ -0,0 +1,19 @@
+package lotto.view
+
+enum class LottoResultMessage(private val message: String) {
+    PROFIT("이득이"),
+    PRINCIPAL("본전이"),
+    LOSS("손해");
+
+    companion object {
+        private const val PRINCIPAL_RATE = 1
+        fun message(rateOfReturn: Float): String {
+            if (rateOfReturn > PRINCIPAL_RATE) {
+                return PROFIT.message
+            }
+            return if (rateOfReturn == PRINCIPAL_RATE.toFloat()) {
+                PRINCIPAL.message
+            } else LOSS.message
```

Comment:
```suggestion
            } else {
                LOSS.message
            }
```
else 문에도 반드시 중괄호를 넣어주시는 것이 컨벤션입니다.

ref. https://kotlinlang.org/docs/coding-conventions.html#control-flow-statements